tests/functional/remote/01-file-install.t

```diff
@@ -15,106 +15,137 @@
     -s "CYLC_TEST_PLATFORM='${CYLC_TEST_PLATFORM}'"
-suite_run_ok "${TEST_NAME_BASE}-run1" cylc run "${SUITE_NAME}" \
+suite_run_ok "${TEST_NAME}-run1" cylc run "${SUITE_NAME}" \
+    --no-detach \
     -s "CYLC_TEST_PLATFORM='${CYLC_TEST_PLATFORM}'"
-RRUND="cylc-run/${SUITE_NAME}"
-poll_grep_suite_log 'Holding all waiting or queued tasks now'
+
+# ensure these files get installed on the remote platform
 SSH="$(cylc get-global-config -i "[platforms][$CYLC_TEST_PLATFORM]ssh command")"
 ${SSH} "${CYLC_TEST_HOST}" \
-find "${RRUND}/"{app,bin,etc,lib} -type f | sort > 'find.out'
+    find "${SUITE_RUN_DIR}/"{app,bin,etc,lib} -type f | sort > 'find.out'
 cmp_ok 'find.out'  <<__OUT__
-${RRUND}/app/moo
-${RRUND}/bin/moo
-${RRUND}/etc/moo
-${RRUND}/lib/moo
+${SUITE_RUN_DIR}/app/moo
+${SUITE_RUN_DIR}/bin/moo
+${SUITE_RUN_DIR}/etc/moo
+${SUITE_RUN_DIR}/lib/moo
 __OUT__
 
-cylc stop --max-polls=60 --interval=1 "${SUITE_NAME}"
 purge
 
-install_suite "${TEST_NAME_BASE}"
+# Test the [scheduler]install configuration
+TEST_NAME="${TEST_NAME_BASE}-configured-paths"
+init_suite "${TEST_NAME}" <<__FLOW_CONFIG__
+[scheduler]
+    install = dir1/, dir2/, file1, file2
+[scheduling]
+    [[graph]]
+        R1 = foo
+[runtime]
+    [[foo]]
+        platform = $CYLC_TEST_PLATFORM
+__FLOW_CONFIG__
+
+create_files
+
+run_ok "${TEST_NAME}-validate" cylc validate "${SUITE_NAME}" \
+    -s "CYLC_TEST_PLATFORM='${CYLC_TEST_PLATFORM}'"
+suite_run_ok "${TEST_NAME}-run2" cylc run "${SUITE_NAME}" \
+    --no-detach \
+    -s "CYLC_TEST_PLATFORM='${CYLC_TEST_PLATFORM}'"
 
-export SECOND_RUN="dir1/, dir2/, file1, file2"
-run_ok "${TEST_NAME_BASE}-validate" cylc validate "${SUITE_NAME}" \
-    -s "CYLC_TEST_PLATFORM='${CYLC_TEST_PLATFORM}'" \
-    -s "SECOND_RUN='${SECOND_RUN}'"
-suite_run_ok "${TEST_NAME_BASE}-run2" cylc run "${SUITE_NAME}" \
-    -s "CYLC_TEST_PLATFORM='${CYLC_TEST_PLATFORM}'" \
-    -s "SECOND_RUN='${SECOND_RUN}'"
-poll_grep_suite_log 'Holding all waiting or queued tasks now'
 ${SSH} "${CYLC_TEST_HOST}" \
-find "${RRUND}/"{app,bin,dir1,dir2,file1,file2,etc,lib} -type f | sort > 'find.out'
+    find "${SUITE_RUN_DIR}/"{app,bin,dir1,dir2,file1,file2,etc,lib} -type f | sort > 'find.out'
 cmp_ok 'find.out'  <<__OUT__
-${RRUND}/app/moo
-${RRUND}/bin/moo
-${RRUND}/dir1/moo
-${RRUND}/dir2/moo
-${RRUND}/etc/moo
-${RRUND}/file1
-${RRUND}/file2
-${RRUND}/lib/moo
+${SUITE_RUN_DIR}/app/moo
+${SUITE_RUN_DIR}/bin/moo
+${SUITE_RUN_DIR}/dir1/moo
+${SUITE_RUN_DIR}/dir2/moo
+${SUITE_RUN_DIR}/etc/moo
+${SUITE_RUN_DIR}/file1
+${SUITE_RUN_DIR}/file2
+${SUITE_RUN_DIR}/lib/moo
 __OUT__
 
-cylc stop --max-polls=60 --interval=1 "${SUITE_NAME}"
 purge
 
+if ! command -v xfs_mkfile; then
+    skip 2
+    exit
+fi
+
 # Test file install completes before dependent tasks are executed
-create_test_global_config "" "
-[platforms]
-    [[cinderella]]
-        hosts = ${CYLC_TEST_HOST}
-        install target = ${CYLC_TEST_INSTALL_TARGET}
```

Comment:
Need to configure platforms using the `TEST_PLATFORM_NAME` in order to inherit any test configuration (e.g. comms mode) as we don't currently have an `inherit` configuration for platforms.